app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -55,7 +57,22 @@ class PatientComponent extends Component {
             // then the ageRangeOnDate should be deleted
             delete searchParameters.ageRangeOnDate;
         }
-
+        // for dead people, diedDuring period -> startDate == now && endDate == the begining of the world
+        // for living people, diedDuring period -> startDate == now && endDate > startDate
+        const today = new Date();
+        const dayFormat = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
+        if (this.state.livingStatus === 'alive') {
+            searchParameters.diedDuringPeriod = [
+                { name: 'endDate', dataType: 'date', value: dayFormat, livingStatus: 'alive' }
```

Comment:
What if you extract the common things in line 66 and 73 into a single statement e.g 
```
searchParameters.diedDuringPeriod = [
    { name: 'endDate', dataType: 'date', value: dayFormat }
}
```
then you can now set the livingStatus property in the condition e.g `searchParameters.diedDuringPeriod[0].livingStatus = 'alive'`